src/components/util/resolve-cell-styles.js

```diff
@@ -1,7 +1,31 @@
-import merge from "lodash.merge";
+/* eslint-disable no-magic-numbers */
 import { mergeStyles } from "radium/lib/merge-styles";
+import Fraction from "fraction.js";
```

Comment:
4.45 KB minified, not sure about gzipped. Originally I needed to do some funky things with fraction denominators, but now that I infer row counts, I'm not sure we even need this. A couple of functions to parse the fraction strings and add those together could be enough.